veracruz-server-sgx/src/server.rs

```diff
@@ -0,0 +1,133 @@
+//! The Veracruz server
```

Comment:
There is very little in this file that is SGX-specific, and this file is likely to appear in all platform-specific servers.
Is there a way that we can use this code across all platforms instead of duplicating it?